tests/data/chrom_size/hg19.chrom.sizes

```diff
@@ -0,0 +1,93 @@
+chr1	249250621
+chr2	243199373
```

Comment:
Can we rename this file to `tests/data/sizes/test.sizes` ?

Reply:
I have changed the name to tests/data/sizes/test.sizes